app/views/start/index.html.slim

```diff
@@ -33,29 +33,16 @@
     h2.govuk-heading-m = t(".service_function")
     - t(".service_function_paragraphs").each
       p.govuk-body = _1
-  - elsif FeatureFlags.enabled?(:partner)
-    |
+
```

Comment:
I think we need and `- else` here. Otherwise we get `p.govuk-body = t(".unacceptable_client_explanation")` twice if `controlled` feature flag is enabled, and zero times if it isn't.